lib/compat/wordpress-6.6/rest-api.php

```diff
@@ -29,3 +29,44 @@ function wp_api_template_access_controller( $args, $post_type ) {
 	}
 }
 add_filter( 'register_post_type_args', 'wp_api_template_access_controller', 10, 2 );
+
+/**
+ * Adds the post classes to the REST API response.
+ *
+ * @param  array  $post  The response object data.
+ *
+ * @return array
+ */
+function gutenberg_add_class_list_to_api_response( $post ) {
+
+	if ( ! isset( $post['id'] ) ) {
+		return array();
+	}
+
+	return get_post_class( array(), $post['id'] );
+}
+
+/**
+ * Adds the post classes to public post types in the REST API.
+ */
+function gutenberg_add_class_list_to_public_post_types() {
+	$post_types = get_post_types( array( 'public' => true ), 'names' );
```

Comment:
I think it makes sense to also add the `show_in_rest=>true` argument here.

Reply:
@ntsekouras I added argument: 'show_in_rest' => true